scanpipe/tests/test_forms.py

```diff
@@ -57,3 +58,22 @@ def test_scanpipe_forms_inputs_base_form_input_urls(self, mock_get):
         self.assertEqual(["archive.zip"], self.project1.input_files)
         expected = {"archive.zip": "https://example.com/archive.zip"}
         self.assertEqual(expected, self.project1.input_sources)
+
+    @mock.patch("requests.get")
+    def test_scanpipe_forms_project_form_name(self, mock_get):
+        data = {
+            "name": "  Test   Name   ",
+            "input_urls": "https://example.com/archipe.zip",
+            "pipeline": "scan_codebase",
+        }
+
+        mock_get.side_effect = None
+        mock_get.return_value = mock.Mock(
+            content=b"\x00",
+            headers={},
+            status_code=200,
+            url="url/archive.zip",
+        )
```

Comment:
There's no need for mock here, we are only testing the name cleanup, you can remove the "input_urls" from the input data to simplify this test.